search_single_spectrum/tools/search_single_spectrum/foodomics_track.py

```diff
@@ -46,18 +46,35 @@ def calculate_enrichment(file_occurrent_df, metadata_df):
 
     return pd.DataFrame(output_list)
 
+def metadata_file_matches(file_occurrent_df):
+    # Filter to appropriate dataset
+    file_occurrent_df = file_occurrent_df[file_occurrent_df["dataset_id"] == "MSV000084900"]
+
+    # Get foodomics full metadata
+    gfop_meta = pd.read_csv('https://raw.githubusercontent.com/ka-west/GFOPontology/master/data/foodomics_metadata_foodmasst.txt', sep='\t')
```

Comment:
We might want to create the data, metadata, code repositories first and link out to those files. 
This is the new reduced file right? Is it only for foodmasst or can we name it just foodomics_metadata.txt

Reply:
Oh yes good point about transferring the repos first. It is only for foodmasst, we want to keep the whole original file as well.

Reply:
maybe name it .tsv to signal that it is tab-separated